Tests/TestUtils/TestBase.swift

```diff
@@ -58,12 +58,12 @@ class TestBase: XCTestCase {
 
     public override func tearDown() {
         super.tearDown()
-
+//        unregisterInstrumentedExtension()
```

Comment:
do we need this change or can we revert?

Reply:
Thanks for checking, changes look good to me. We can revisit this in the future and see if we can remove the unused API and the associated code in the InstrumentedExtension.